packages/devtools_app/lib/src/standalone_ui/vs_code/property_editor_panel.dart

```diff
@@ -39,29 +37,28 @@ class _PropertyEditorSidebarPanelState
     super.initState();
 
     final editor = EditorClient(widget.dtd);
-    ga.screen(PropertyEditorSidebarPanel.id);
+    ga.screen(PropertyEditorEvents.id);
     unawaited(_editor = editor.initialized.then((_) => editor));
   }
 
   @override
   Widget build(BuildContext context) {
-    return Column(
-      children: [
-        Expanded(
-          child: FutureBuilder(
-            future: _editor,
-            builder:
-                (context, snapshot) => switch ((
-                  snapshot.connectionState,
-                  snapshot.data,
-                )) {
-                  (ConnectionState.done, final editor?) =>
-                    _PropertyEditorConnectedPanel(editor),
-                  _ => const CenteredCircularProgressIndicator(),
-                },
-          ),
+    return Align(
+      alignment: Alignment.topCenter,
+      child: Expanded(
```

Comment:
Expanded should only be used as a direct child of a flex widget like Column or Row

Reply:
Oops thanks, removed the expanded!